src/components/ApplicationGroup/AppGroupAppFilter.components.tsx

```diff
@@ -25,7 +25,7 @@ export const ValueContainer = (props): JSX.Element => {
     } else {
         selectorText = `${selectedAppList.length > 0 ? selectedAppList.length : appListOptions.length}/${
             appListOptions.length
-        } Applications`
+        } ${filterParentType===FilterParentType.env? 'Applications': 'Environments'}`
```

Comment:
Please format this line

Reply:
if filterParentType===FilterParentType.env then are we showing the Application?

Reply:
Done